README.md

```diff
@@ -143,6 +143,26 @@ index 2878332..1f3103a 100644
 +CMD [ "/usr/bin/python3" ]
 ```
 
+# How to use Claude AI
+
+To access Claude AI, you need an API key. Follow these steps to get started:
+
+1. Create an account
+- Visit [Anthropic Console](https://console.anthropic.com/login) and sign up for an account. 
+
+2. Get your API key
+- Once logged in, navigate to the API settings and generate your API key.
+
+3. Log in with your API key
+- Insert your API key when prompted to authenticate your access to Claude AI.
```

Comment:
We need to be much more detailed in how we explain how to use our UI

It's really misleading to say "prompted to authenticate". There is a prompt as in a text box, but there is no explicit request or popup which is want "prompted" usually implies.

Add pictures for the whole process.